reactive-client/src/ios/Default-Info.plist

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+        <key>CFBundleIdentifier</key>
+        <string>org.gluonhq.reactive.RxGluonApp</string>
+        <key>CFBundleVersion</key>
+        <string>1.0</string>
+        <key>CFBundleExecutable</key>
+        <string>RxGluonApp</string>
```

Comment:
Wrong name. It should be `reactive-client`, as the folder's project name. This is preventing deploying it on an iOS device.

Reply:
Updated